modules/460-log-shipper/hooks/internal/vector/destination/common.go

```diff
@@ -34,11 +34,12 @@ type Healthcheck struct {
 }
 
 type CommonTLS struct {
-	CAFile         string `json:"ca_file,omitempty"`
-	CertFile       string `json:"crt_file,omitempty"`
-	KeyFile        string `json:"key_file,omitempty"`
-	KeyPass        string `json:"key_pass,omitempty"`
-	VerifyHostname bool   `json:"verify_hostname"`
+	CAFile            string `json:"ca_file,omitempty"`
+	CertFile          string `json:"crt_file,omitempty"`
+	KeyFile           string `json:"key_file,omitempty"`
+	KeyPass           string `json:"key_pass,omitempty"`
+	VerifyHostname    bool   `json:"verify_hostname"`
+	VerifyCertificate bool   `json:"verify_certificate"`
 }
```

Comment:
You have `CommonTLSSpec` struct and `CommonTLS` which are equal except the `VerifyX` fields pointer. Is it possible to use single struct?

Reply:
Some legacy stuff. Did not touch it.